contributing_to_docs/doc_guidelines.adoc

```diff
@@ -229,7 +229,9 @@ link:https://redhat-documentation.github.io/modular-docs/#forming-assemblies[Red
 
 [NOTE]
 ====
-When using "Prerequisites", "Next steps", and "Additional resources" headings in an assembly, use `==` formatting, such as `== Prerequisites` and `== Additional resources`. Use of this heading syntax at the assembly level indicates that the sections relate to the whole assembly. Only use `.` syntax (`.Additional resources`) to follow the module as a trailing include at the assembly level, which indicates that the resources apply specifically to the module.
+When using "Prerequisites", "Next steps", and "Additional resources" headings in an assembly, use `==` formatting, such as `== Prerequisites` and `== Additional resources`. Use of this heading syntax at the assembly level indicates that the sections relate to the whole assembly.
+
+Only use `.` syntax (`.Additional resources`) to follow a module as a trailing include at the assembly level, where the formatting indicates that the resources apply specifically to the module. Because you cannot use xrefs in modules, be sure to also include the link as part of the subsection in the assembly.
```

Comment:
Not sure what you mean by _trailing include_.

Reply:
Thanks for calling it out, @mburke. I didn't know either the first time I heard the term. I've rewritten this section to explain better. Do you think this makes sense now?